chatmm/src/main/java/com/likeminds/chatmm/chatroom/detail/model/MemberViewData.kt

```diff
@@ -1,11 +1,11 @@
-package com.likeminds.chatmm.chatroom.model
+package com.likeminds.chatmm.chatroom.detail.model
```

Comment:
why shifted inside `detail`, Also it should be inside chatroom package, move to a different package like member or user